### PR TITLE
Working with Windows

### PR DESCRIPTION
**Pull request by jycouet**
Refer to https://github.com/singapore/renovate/issues/1148
Like this, prettier works well and jest also!

For tests, I found this way of mocking path.join.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -75,13 +75,14 @@
...
-    "yarn": "1.3.2",
-    "vso-node-api": "6.2.8-preview"
+    "vso-node-api": "6.2.8-preview",
+    "yarn": "1.3.2"
   },
   "devDependencies": {
     "babel-plugin-transform-object-rest-spread": "6.26.0",
     "chai": "4.1.2",
     "condition-circle": "1.5.0",
+    "cross-env": "^5.1.1",
```
Comment (rarkins) [inline package.json]: Please remove ^

**Comment by rarkins** [comment]
I think the spyOn solution will make tests harder to maintain in future and so I would like to make sure there aren't any "cleaner" solutions available before we merge this one.

**Comment by jycouet** [comment]
I agree with you, I'm not a big fan of spyOn like this.
It's the only way I found.

Maybe other test should be sufficient? Like split by '/' and '\\' should be the same.

**Comment by rarkins** [comment]
What if you just remove the snapshots for those tests instead of adding spyOn? None of them look essential as we check other things anyway

**Comment by jycouet** [comment]
Fully agree! :)

**Comment by rarkins** [comment]
@jycouet I've made the edits myself, can you git pull and try the tests locally?

**Comment by jycouet** [comment]
Thx @rarkins, it's working.
I updated  snapshots, now it working well 👍

**Comment by rarkins** [comment]
lgtm once CircleCI tests pass

**Pull request merged**